src/test/scala/utils/KafkaConfig.scala

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import cats.Show
+import cats.effect.Sync
+import cats.syntax.all.*
+import org.apache.kafka.common.security.auth.SecurityProtocol
+
+final case class KafkaConfig(
+    protocol: SecurityProtocol,
+    host: String,
+    kafkaPort: Int,
```

Comment:
Haven't checked if they are used elsewhere, but if there are only used to construct `plaintextListener`, how about:
```suggestion
    private val protocol: SecurityProtocol,
    private val host: String,
    private val kafkaPort: Int,
```
to limit their direct usage?

Reply:
It was, but unnecessarily, the plaintextListener could be used instead
[`94ea403` (#109)](https://github.com/sky-uk/fs2-kafka-topic-loader/pull/109/commits/94ea403f6da846e25cdd9c0e0731204c81df28d4)